desk/app/notify.hoon

```diff
@@ -64,9 +64,11 @@
       pass  pass:io
   ::
   ++  on-init
+    =/  tomorrow=@da  (add now.bowl ~d1)
     :_  this
     :~  (~(watch-our pass:io /hark) %hark /ui)
         (~(wait pass:io /clear) (add now.bowl clear-interval))
+        (~(wait pass:io /daily-timer) tomorrow)
```

Comment:
we need to also do this in on-load since most people will already have this agent installed. just need to make sure we don't do both. I'm also wondering if we should do the provider check here instead of in the the thread.

Reply:
move it to a poke and then have the agent poke itself, in other words?

Reply:
yeah move to poke, and then when you receive the poke check to see if we're already waiting, and then have both init and load do the poke.